application/email/v0/config/tasks.json

```diff
@@ -0,0 +1,329 @@
+{
+  "$defs": {
+    "search-subject": {
+      "description": "Search for emails with a specific subject",
+      "instillAcceptFormats": [
+        "string"
+      ],
+      "instillUpstreamTypes": [
+        "value",
+        "reference",
+        "template"
+      ],
+      "instillUIOrder": 4,
+      "title": "Search Subject",
+      "type": "string"
+    },
+    "search-from": {
+      "description": "Search for emails from a specific email address",
+      "instillAcceptFormats": [
+        "string"
+      ],
+      "instillUpstreamTypes": [
+        "value",
+        "reference",
+        "template"
+      ],
+      "instillUIOrder": 2,
+      "title": "Search From",
+      "type": "string"
+    },
+    "search-to": {
+      "description": "Search for emails to a specific email address",
+      "instillAcceptFormats": [
+        "string"
+      ],
+      "instillUpstreamTypes": [
+        "value",
+        "reference",
+        "template"
+      ],
+      "instillUIOrder": 3,
+      "title": "Search To",
+      "type": "string"
+    },
+    "limit": {
+      "description": "The maximum number of emails to search for",
+      "instillAcceptFormats": [
+        "integer"
+      ],
+      "instillUpstreamTypes": [
+        "value",
+        "reference"
+      ],
+      "instillUIOrder": 5,
+      "title": "Limit",
+      "type": "integer"
+    },
+    "date": {
+      "description": "Search for emails with the date the email was sent",
+      "instillFormat": "string",
+      "instillAcceptFormats": [
+        "string"
+      ],
+      "instillUpstreamTypes": [
+        "value",
+        "reference",
+        "template"
+      ],
+      "instillUIOrder": 6,
+      "title": "Search Date",
+      "type": "string"
+    },
+    "search-email-message": {
+      "description": "Search for emails with a specific message",
+      "instillAcceptFormats": [
+        "string"
+      ],
+      "instillUpstreamTypes": [
+        "value",
+        "reference",
+        "template"
+      ],
+      "instillUIOrder": 7,
+      "title": "Search Message",
+      "type": "string"
+    }
+  },
+  "TASK_SEND_EMAILS": {
```

Comment:
Since we only send one email, how about removing the "s"?

Reply:
Ah, I thought an email is like a message to a recipient.
The emails mean a message to several recipients.

I set it as [recipients](https://github.com/instill-ai/component/pull/178/files#diff-f5135e1e71de2a32c9a05b4a2876cb66e90216e330f0fdbcebc2d6f81b6b1079R94)

So, I use plural here.
Should I change it to singular?

Reply:
'Send emails' sounds more like sending different content, but in this case, you're sending just one.